taskt/Core/Automation/Commands/Dictionary Commands/GetDictionaryValueCommand.cs

```diff
@@ -1,110 +1,115 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
+using System.Collections.Generic;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using System.Collections.Generic;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Dictionary Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to loop through an Excel Dataset")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to iterate over a series of Excel cells.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command attempts to loop through a known Excel DataSet")]
+    [Group("Dictionary Commands")]
+    [Description("This command allows user to get a dictionary value based on a specified key.")]
     public class GetDictionaryValueCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the output variable")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a unique dataset name that will be used later to traverse over the data")]
-        [Attributes.PropertyAttributes.SampleUsage("**myData**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_OutputVariable { get; set; }
+        [PropertyDescription("Dictionary Name")]
```

Comment:
I think "Name" is misleading here, since it's not the name you're inputting, it's a variable containing the dictionary itself. Same goes for Key Name below

Reply:
done